package.json

```diff
@@ -60,6 +60,7 @@
 		"electron-is-dev": "^3.0.1",
 		"electron-squirrel-startup": "^1.0.1",
 		"electron-store": "^10.0.0",
+		"maplibre-gl": "^4.7.1",
```

Comment:
although very new, will highlight that v5 was released last week

https://github.com/maplibre/maplibre-gl-js/releases/tag/v5.0.0

although I guess if we use a react wrapper, that should dictate the version we use. most likely v4 for now but if it supports v5, great!